corehq/apps/hqwebapp/management/commands/build_bootstrap5_diffs.py

```diff
@@ -135,6 +142,13 @@ def add_arguments(self, parser):
 
     def handle(self, *args, **options):
         update_app = options.get('update_app')
+
+        if not has_no_pending_git_changes():
```

Comment:
I think this is too strict. Warning the user of open changes is useful, but they should be allowed to decide to proceed. For one, it'll make it harder to troubleshoot/develop this command itself, since you can't test experimental changes to the command, you have to commit them.

Reply:
updated

Reply:
thanks!